assets/sass/protocol/base/elements/_links.scss

```diff
@@ -8,8 +8,11 @@
     color: $link-color;
     text-decoration: underline;
 
+    &:visited {
```

Comment:
`:visited` moves earlier in the source so is over-ridden by the hover, focus, and active styles that follow. We rely on the cascade to style a link when it's hovered instead of explicitly declaring :visited:hover styles.